torch/nn/modules/module.py

```diff
@@ -556,7 +556,7 @@ def get_extra_state(self) -> Any:
         """
         raise RuntimeError(
             "Reached a code path in Module.get_extra_state() that should never be called. "
-            "Please file an issue at https://github.com/pytorch/pytorch/issues/new?template=bug-report.md "
+            "Please file an issue at https://github.com/pytorch/pytorch/issues/new?template=bug-report.yml "
```

Comment:
unfortunately by helping updating this part, this PR won't be mergeable by bot ... Leaving for you to decide

Reply:
I will create a separate PR then